packages/cli-plugin-deploy-pulumi/commands/watch/watchPackages.js

```diff
@@ -96,7 +98,7 @@ module.exports = async ({ inputs, output, context }) => {
 
                 worker.postMessage(
                     JSON.stringify({
-                        options: { env, debug, logs },
+                        options: { env, debug, logs: !multipleWatches },
```

Comment:
Let's make this `logs: !multipleWatches || logs`.

In other words, show full compilation logs if we're only watching a single package (`!multipleWatches`) or if specifically instructed by the user (`|| logs`).